src/credentials/providers/ssoCredentialProvider.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { Credentials, SSO } from 'aws-sdk'
+import { getLogger } from '../../shared/logger'
+import { SsoAccessTokenProvider } from '../sso/ssoAccessTokenProvider'
+import * as nls from 'vscode-nls'
+
+const localize = nls.loadMessageBundle()
+
+export class SsoCredentialProvider {
```

Comment:
Why doesn't `SsoCredentialProvider` implement the `CredentialsProvider` interface?